azurerm/internal/services/devspace/resource_arm_devspace_controller.go

```diff
@@ -47,25 +47,37 @@ func resourceArmDevSpaceController() *schema.Resource {
 
 			"resource_group_name": azure.SchemaResourceGroupName(),
 
+			"sku_name": {
+				Type:     schema.TypeString,
+				Optional: true, // required in 2.0
+				Computed: true, // remove in 2.0
+				//ForceNew:      true, // uncomment in 2.0 - this should be fine as there is only 1 valid value
+				ConflictsWith: []string{"sku"},
+				ValidateFunc: validation.StringInSlice([]string{
+					"S1",
+				}, false),
+			},
+
 			"sku": {
 				Type:     schema.TypeList,
-				Required: true,
-				ForceNew: true,
-				MaxItems: 1,
+				Optional: true,
+				//ForceNew:      true, // uncomment in 2.0 - this should be fine as there is only 1 valid value
+				Computed:      true,
+				ConflictsWith: []string{"sku_name"},
+				Deprecated:    "This property has been deprecated in favour of the 'sku_name' property and will be removed in version 2.0 of the provider",
+				MaxItems:      1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"name": {
 							Type:     schema.TypeString,
 							Required: true,
-							ForceNew: true,
 							ValidateFunc: validation.StringInSlice([]string{
 								"S1",
```

Comment:
For consistency with the other `sku_name`s should we make this `S_1`?

Reply:
I don't want to make confusing alternatives, its up to azure to change it